### PR TITLE
Add am layout for Corne (crkbd)

### PR DESCRIPTION
**Pull request by am**
## Description

Adding the layout I use for a Corne keyboard. A few features that can be found:
- QWERTY
- Number layer: with row and pads(normal and inverted)
- Symbols + Fns layer
- Adjust layer: Reset power and media keys (OSX)
- Touchcursor layer: VIM style for cursos keys and mouse

- Some tapdance for cmd / alt on a single key + cmd + alt on another

IMG_4431

😺 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/keymaps/am/config.h [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+//#define USE_MATRIX_I2C
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
+
+// https://medium.com/@keebio/the-case-of-the-wayward-elite-c-73f0fd691f88
+#define SPLIT_USB_DETECT
+
+#define SSD1306OLED
```
[the remark this answers is not in the document]
Comment (am) [reply keyboards/crkbd/keymaps/am/config.h:32]: Thank you @drashna for the review! I had no idea! If the only problem is the timeout I could not see any problem. It turns off correctly and turns on on keypress. Any other issue I should consider?

File: keyboards/crkbd/keymaps/am/keymap.c [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+  }
+}
+
+void iota_gfx_task_user(void) {
+  struct CharacterMatrix matrix;
+  matrix_clear(&matrix);
+  matrix_render_user(&matrix);
+  matrix_update(&display, &matrix);
+}
+#endif//SSD1306OLED
```
[the remark this answers is not in the document]
Comment (am) [reply keyboards/crkbd/keymaps/am/keymap.c:180]: I'll address this. Thank you. Great to have this reviewed ;)

[the remark this answers is not in the document]
Comment (am) [reply keyboards/crkbd/keymaps/am/keymap.c:180]: @drashna I've been playing with the `OLED_DRIVER` but couldn't find any example for a keylogger. Currently, I'm able to print the modes, but nothing more. Do you know how any example you could point me to? Thank you ;)

[the remark this answers is not in the document]
Comment (drashna) [reply keyboards/crkbd/keymaps/am/keymap.c:180]: Of course!
https://github.com/qmk/qmk_firmware/blob/1a159a38ed127fa80762a7c1b8150ba19ac34ddf/users/drashna/oled_stuff.c#L5-L76

File: keyboards/crkbd/keymaps/am/config.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (drashna) [reply keyboards/crkbd/keymaps/am/config.h:32]: It supports a lot more functionality, such as rotation, for instance.

**Pull request closed without merging**